pages/content/amp-dev/documentation/guides-and-tutorials/develop/custom-javascript.md

```diff
@@ -0,0 +1,139 @@
+---
+$title: Include Custom JavaScript in AMP Pages
+$order: 7
+formats:
+  - websites
+author: CrystalOnScript
+contributors:
+  - fstanis
+---
+
+AMP strives to provide a consistently good experience to all users across the web by encouraging the use of high-functioning and seamless components that are ready to go out of the box. 
+
+Some web experiences require a high amount of customization that go beyond the state binding capabilities of [`amp-bind`](https://amp.dev/documentation/components/reference/amp-bind.html?format=websites) and the dynamic data retrieval and templating functionality of [`amp-list`](https://amp.dev/documentation/components/reference/amp-list.html?format=websites) and [`amp-mustache`](https://amp.dev/documentation/components/reference/amp-mustache.html?format=websites). For those one-of-a-kind cases, AMP has created [`<amp-script>`](https://amp.dev/documentation/components/reference/amp-script.html?format=websites), a component that allows the use of arbitrary JavaScript on your AMP page without affecting the page's overall performance.
```

Comment:
Perhaps "one of a kind" is too extreme. Could we soften this?

Reply:
changing to "one-off", sounds a little softer